modules/context/base.go

```diff
@@ -198,20 +198,6 @@ func (b *Base) FormInt64(key string) int64 {
 	return v
 }
 
-// FormInt64s returns a int64 slice for the provided key from the form
-func (b *Base) FormInt64s(key string) (v []int64) {
-	str := b.FormStrings(key)
-	if str == nil {
-		return nil
-	}
-
-	for _, s := range str {
-		i, _ := strconv.ParseInt(s, 10, 64)
-		v = append(v, i)
-	}
-	return v
-}
-
```

Comment:
This is a breaking change. `FormInt64s` may be shared among many places.

Reply:
<img width="694" alt="image" src="https://github.com/go-gitea/gitea/assets/80308335/e2636047-fafe-48bc-ba1b-5b2bf9c8aae1">

Reply:
`FormInt64` still exists, `FormInt64s` was added in https://github.com/go-gitea/gitea/pull/24212/commits/e50b52ccd08ab7cbe7e7691d7c86da5a59551864#diff-fa4a8538493d18cf504db405186f872622f67001de4bd2ecb5807d7e5408ccfd as a int64 counterpart for `FormStrings` in this PR and was only used for fetching this ids in the api. But it has since been removed since the ids was replaced by proper user/org support in the last commit :)